internal/pkg/coordinator/monitor.go

```diff
@@ -434,7 +434,7 @@ func runUnenroller(ctx context.Context, bulker bulk.Bulk, policyId string, unenr
 		Dur("checkInterval", checkInterval).
 		Dur("unenrollTimeout", unenrollTimeout).
 		Msg("unenroll monitor start")
-	defer l.Info().Msg("unenroll monitor exit")
+	defer l.Info().Msg("Unenroll monitor exit")
```

Comment:
Are we capitalizing log messages now?  I wish folks would make up their mind.

Reply:
I see, we are not consistent at all. The way I think of it is when I see an log message is that it should be a sentence I read. Sentences start with an uppercase letter. I think that is the way we did it mostly in Beats. But this is total personal preference. Looking at the fleet-server code, we don't do it.

What is your preference and why?

Reply:
Honestly don't care.  I prefer starting with a capital letter.  Blake didn't do it that way and  wanted lower case.  I just ask that we are consistent.